new-packages/ts-project/__tests__/utils.ts

```diff
@@ -149,6 +150,12 @@ export function replaceUniqueIds(
       ...Object.keys(digraph.blocks).map(
         (id, i) => [id, `block-${i}`] as const,
       ),
+      ...Object.values(digraph.blocks)
+        .filter((block): block is ActorBlock => block.blockType === 'actor')
+        .filter((block) => block.properties.id.startsWith('inline:'))
+        .map(
+          (block, i) => [block.properties.id, `inline:actor-id-${i}`] as const,
+        ),
```

Comment:
nit: this could be merged with the existing iteration over blocks with the help of `.flatMap`. It's a test code so whatever ;p